terraform/env_types/gcp/gke/legion/variables.tf

```diff
@@ -58,8 +58,11 @@ variable "docker_user" {
 variable "docker_password" {
   description = "Legion Docker repo password"
 }
-variable "enclave_jwt_secret" {
-  description = "Legion enclave JWT secret"
+variable "api_private_key" {
+  description = "RSA model private key"
```

Comment:
Please specify purpose of this key (e.g. `is used for ...`)

Reply:
Fixed